src/mixins/java/org/spongepowered/common/mixin/api/minecraft/world/level/levelgen/NoiseSettingsMixin_API.java

```diff
@@ -49,8 +50,10 @@
     @Shadow public abstract int shadow$noiseSizeHorizontal();
     @Shadow public abstract int shadow$noiseSizeVertical();
     @Shadow @Deprecated public abstract boolean shadow$isAmplified();
```

Comment:
if amplified is deprecated, should we axe from API? Is there a replacement?

Reply:
yeah its probably `Shaper`/`TerrainShaper`